### PR TITLE
pls dont touch

### PR DESCRIPTION
**Pull request by hmoco**

### CODE REVIEW

File: addon/models/relationship.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
+    target_type: DS.attr('string'),
```
Comment (abought) [inline addon/models/relationship.js]: In general, try to follow JS conventions for names where possible (`targetType` vs `target_type`). Eg, the application adapter should take care of converting names from camelcase to snakecase and back, in cases where the API response does not match ember data conventions.

I wish I could say I was being pedantic, but Ember has a *lot* of magical assumptions based on how fields are named.

**Comment by abought** [comment]
PR description would be helpful.

File: addon/models/relationship.js (hunk 1 above)
Comment (samchrisinger) [inline addon/models/relationship.js]: In particular, the way the ApplicationSerializer transforms these fields will probably break this (underscored -> camelized)

File: addon/serializers/application.js [hunk 2]
```diff
@@ -32,6 +32,9 @@ export default DS.JSONAPISerializer.extend({
             return Ember.String.camelize(key);
         }
     },
+    keyForRelationship(key) {
```
Comment (samchrisinger) [inline addon/serializers/application.js]: This probably needs to borrow the pattern used in keyForAttribute (where serialize applies the inverse transformation of deserialize)

**Pull request closed without merging**